lighthouse-core/report/html/templates.html

```diff
@@ -310,7 +304,12 @@
     <a href="" class="lh-topbar__url" target="_blank" rel="noopener"></a>
     
     <div class="lh-export">
-      <button class="report-icon report-icon--share lh-export__button" title="Export report"></button>
+      <div class="report-icon report-icon--share lh-export__button" title="Export report">
+        <svg width="100%" height="100%" viewBox="0 0 24 24" fill="#000000">
```

Comment:
How about using grey-600 instead of #000000 and opacity 0.54?
```suggestion
        <svg width="100%" height="100%" viewBox="0 0 24 24" fill="#757575">
```

Reply:
or even better, can we set the fill with our CSS variable :)

Reply:
done